plugins/ssl/letsencrypt_weekly

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+: << =cut
+=head1 NAME
+
+letsencrypt_weekly - monitor the number of CSRs by week for /etc/letsencrypt/csr/
+
+see https://letsencrypt.org/docs/rate-limits/
+
+= head1 CONFIGURATION
+
+There are no special configuration options for this plugin
+
+=head1 Dependencies
+
+Dependencies: openssl
+
+=head1 AUTHOR
+
+andreas perhab - andreas.perhab@wt-io-it.at
+https://www.wt-io-it.at/
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 MAGIC MARKERS
+
+ #%# family=auto
+ #%# capabilities=autoconf
+
+=cut
+
+. "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+warning=${warning:-:40}
+
+if [ "$1" = "autoconf" ] ; then
+    find /etc/letsencrypt/csr/ > /dev/null 2>&1 && echo "yes" || echo "no (no files below /etc/letsencrypt/csr)"
```

Comment:
At the moment the reason does not really match the test. Maybe use the following?
```shell
[ -d /etc/letsencrypt/csr/ ] && echo "yes" || echo "no (directory /etc/letsencrypt/csr does not exist)"
```

Reply:
changed to a test of existing directory (better than checking if there is already a CSR)